tests/suites/deploy/deploy_bundles.sh

```diff
@@ -54,22 +54,63 @@ run_deploy_cmr_bundle() {
 	destroy_model "other"
 }
 
-run_deploy_exported_bundle() {
+# run_deploy_exported_charmstore_bundle_with_fixed_revisions is to test which checks
+# how juju deploys charmstore bundles with fixed revisions
+run_deploy_exported_charmstore_bundle_with_fixed_revisions() {
 	echo
 
-	file="${TEST_DIR}/test-export-bundles-deploy.log"
+	file="${TEST_DIR}/test-export-bundles-deploy-with-fixed-revisions.log"
 
-	ensure "test-export-bundles-deploy" "${file}"
+	ensure "test-export-bundles-deploy-with-fixed-revisions" "${file}"
 
 	bundle=./tests/suites/deploy/bundles/telegraf_bundle.yaml
```

Comment:
ok, and for 2.9 this *does* have pinned revisions, so it fits. Presumably you'll update this test for landing it on 3.0?

Reply:
Yes, If I understand you correctly, we will need to update (and add revisions) to telegraf_bundle.yaml on 3.0, too